### PR TITLE
Integration Tests HAL-01 Verification Fix

### PR DESCRIPTION
**Pull request by Dom-Roth**
## Issue
During a meeting today we realized a bug i made within the integration tests for HAL-01.
Instead of checking the oracle stake getting slashed, I checked the wallet balance. Which is what HAL-02 was all about.

### CODE REVIEW

**Review by Juliansv — APPROVED** [review]

**Review by kollegian — DISMISSED** [review]
LGTM

**Comment by Dom-Roth** [comment]
bors ping

File: integration-tests/runtime-tests/test/tests/oracle/HAL01-Tests.ts [hunk 1]
```diff
@@ -224,17 +224,17 @@ describe("HAL01 [Oracle] Tests", function () {
       const maliciousPrice = api.createType("u128", 900);
       const asset = api.createType("u128", assetID);
 
-      const balanceWallet1BeforeTransaction = new BN(
-        (await api.rpc.assets.balanceOf(asset.toString(), walletHAL01_1.publicKey)).toString()
+      const oracleStakeWallet1BeforeTransaction = new BN(
+        (await api.query.oracle.oracleStake(walletHAL01_1.publicKey)).toString()
```
Comment (andor0) [inline integration-tests/runtime-tests/test/tests/oracle/HAL01-Tests.ts:228]: looks like, a helper-function will useful
```
const oracleStake = async (wallet: KeyringPair) => {
  return new BN(
    (await api.query.oracle.oracleStake(wallet.publicKey)).toString()
   );
}
```
then we can simplify
```
const oracleStakeWallet1BeforeTransaction = await oracleStake(walletHAL01_1);
const oracleStakeWallet2BeforeTransaction = await oracleStake(walletHAL01_2); 
// ...

const oracleStakeWallet1AfterTransaction = await oracleStake(walletHAL01_1);
const oracleStakeWallet2AfterTransaction = await oracleStake(walletHAL01_2);
// ...

```

Comment (Dom-Roth) [reply integration-tests/runtime-tests/test/tests/oracle/HAL01-Tests.ts:228]: Makes sense, will update this. 👍 
Thanks a lot. 🙏

**Review by andor0 — APPROVED** [review]

**Review by kollegian — APPROVED** [review]

**Pull request merged**